integration/boofcv-WebcamCapture/examples/boofcv/examples/ExampleWebcamObjectTrackingGst.java

```diff
@@ -0,0 +1,231 @@
+/*
+ * Copyright (c) 2020, Peter Abeles. All Rights Reserved.
+ *
+ * This file is part of BoofCV (http://boofcv.org).
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package boofcv.examples;
+
+import boofcv.abst.tracker.TrackerObjectQuad;
+import boofcv.factory.tracker.FactoryTrackerObjectQuad;
+import boofcv.io.gstwebcamcapture.UtilWebcamCapture;
+import boofcv.io.gstwebcamcapture.Webcam;
+import boofcv.io.image.ConvertBufferedImage;
+import boofcv.struct.image.GrayU8;
+import boofcv.struct.image.ImageBase;
+import boofcv.struct.image.ImageType;
+import boofcv.struct.image.Planar;
+import georegression.geometry.UtilPolygons2D_F64;
+import georegression.struct.point.Point2D_I32;
+import georegression.struct.shapes.Quadrilateral_F64;
+import georegression.struct.shapes.Rectangle2D_F64;
+
+import javax.swing.*;
+import java.awt.*;
+import java.awt.event.MouseEvent;
+import java.awt.event.MouseListener;
+import java.awt.event.MouseMotionListener;
+import java.awt.image.BufferedImage;
+
+/**
+ * Example of how to open a webcam and track a user selected object.  Click and drag the mouse
+ * to select an object to track.
+ *
+ * @author Peter Abeles
+ */
+public class ExampleWebcamObjectTrackingGst<T extends ImageBase<T>> extends JPanel
```

Comment:
Let's keep the usage example to be very simple. just opening a webcam and displaying the stream in a window for now.